### ISSUE TITLE
Fields Hints are not hiding after being selected

### ISSUE THREAD

**Issue by dexter29498** [graphql/graphql-playground#1430]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [ ] GraphQL Playground HTML
- [X] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?
- Happening on every OS. Run in Windows, Linux, Arch

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?
I am using rust packages
- actix-web = "4.8.0"
- juniper = "0.16.1"
- juniper_actix = "0.6.0"

But it's not only happening in rust It's also happening in nodejs package as well.

#### What is the expected behavior?
- It should get hidden after being selected

#### What is the actual behavior?
- Tips are staying on the UI after refresh it hides

#### What steps may we take to reproduce the behavior?
- Just write down the query or mutation it will show the tips and once selected it will stay on UI

_Please provide a gif or image of the issue for a quicker response/fix._
![image](https://github.com/user-attachments/assets/ee8a5b64-b9e3-41fd-ace0-01e652b803e4)

**Comment by asahi-kawajiri** [comment]
I've faced same issue. \(x_x )/

**Comment by dexter29498** [comment]
@asahi-kawajiri  Did you find any solution or atleast workaround?

**Comment by iatanas0v** [comment]
For now you can use Firefox, the issue is related to Chrome deprecating `DOMNodeRemoved` mutation event

**Comment by JoaoBGusmao** [comment]
See https://github.com/graphql/graphql-playground/issues/1429#issuecomment-2274843437

**Comment by scaraux** [comment]
https://github.com/graphql/graphql-playground/pull/1431

**Comment by vinhnglx** [comment]
When do we merge this fix? For now, either Safari or Firefox will help. Chrome or Edge will face the same issue.

**Comment by JoaoBGusmao** [comment]
> When do we merge this fix? For now, either Safari or Firefox will help. Chrome or Edge will face the same issue.

@vinhnglx I don't believe a fix will ever get merged. It is an abandoned project. If you cannot switch to another Graphql IDE, your best option is to fork, apply [a fix](https://github.com/JoaoBGusmao/graphql-playground/commit/52159b9a332558f8996f152efa8d0c030f0db169) and publish it just like I did.
I'm using this playground from a Google Chrome extension, so I published my fork to Chrome Store, [here is the link](https://chromewebstore.google.com/detail/graphql-playground-chrome/gmehhhfompinmcoabokhckceachkjmha) if anyone needs a working Chrome extension.
